helpers/format-website.js

```diff
@@ -0,0 +1,5 @@
+function formatWebsite(website) {
+  return website.startsWith('http') ? website : 'http://' + website;
```

Comment:
I missed calling the `formatWebsite` helper on the websites in the restaurant list, which is used on the homepage and on the hit list page. Nice catch, and nice job tracking it down.